quipucords/quipucords/templates/registration/login.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+
+{% block page-title %}
+Red Hat Entitlements Reporting
```

Comment:
Had my fingers crossed 🤞 **_Scanny McScanny Face_**.

Reply:
lol, localization for the win =)